docs/update-privacy-policy.md

```diff
@@ -0,0 +1,47 @@
+
+# Updating the Privacy Policy
+
+The [Thunderbird Privacy Policy](https://www.thunderbird.net/en-US/privacy/) is automatically generated and should not be edited by hand.
+
+To update it, you will:
+1. Create a new branch.
+2. Run the build tool with the `--downloadlegal` option.
+3. Commit the changes.
+3. Create a Pull Request.
+
+
+Here is an example workflow.
+
+
+## Pull the latest and create a new branch
+
+Assuming you have already cloned the `thunderbird-website` repo:
+
+```bash
+git checkout master
+git pull
+git checkout -b chore/update-legal
+```
+
+## Download the latest legal documents
+
+In order to run the `build-site.py` script, follow the basic setup instructions for thunderbird-website found in the main [README.md](../README.md##Dependencies).
```

Comment:
I know we don't have a structured docs right now, so thank you for adding to it!

But we should assume they have the site setup, along with a setup.md or setup.rst linked in the sidebar incase they dont (handled by index.md, or index.rst). These instructions could change and if we have this step everywhere it means we have to update it everywhere.

Reply:
Good point!
I can copy the setup info from the README.md into a `setup.md` file and add an entry to the `index.rst`

I feel like that should go in its own PR, along with a ticket to update mentions of (or links to) the setup procedure.

Reply:
Created #727